documentation/config_file_master.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: master.conf"
+canonical: "/puppetserver/latest/config_file_master.html"
+---
+
+[`trapperkeeper-authorization`]: https://github.com/puppetlabs/trapperkeeper-authorization
+[new `auth.conf`]: ./conf_file_auth.html
+[Puppet `auth.conf`]: /puppet/latest/reference/config_file_auth.html
+[deprecated]: ./deprecated_features.html
+[`puppetserver.conf`]: ./conf_file_puppetserver.html
+
+The `master.conf` file configures how Puppet Server handles legacy authorization methods for master endpoints. For a broader overview of Puppet Server configuration, see the [configuration documentation](./configuration.html).
+
+> **Deprecation Note:** This file only supports the `allow-header-cert-info` parameter, and is deprecated as of Puppet Server 2.2 in favor of new authorization methods configured in the [new `auth.conf`][] file. Since this configuration file is deprecated, Puppet Server no longer includes a default `master.conf` file in the Puppet Server package.
+
+* In `master.conf`, `allow-header-cert-info` determines whether Puppet Server should use authorization info from the `X-Client-Verify`, `X-Client-CN`, and `X-Client-Cert` HTTP headers. Defaults to `false`.
```

Comment:
All of the references in this doc to `X-Client-CN` should actually be `X-Client-DN`.  Nothing you introduced; we must have missed that in the current docs as well.  The CN is one of the attributes that we extract from the DN.  The expectation, though, is that requests are providing a full DN format for this header value.